### ISSUE TITLE
Google Calendar Attachments

### ISSUE THREAD

**Issue by pearcemichal** [fullcalendar/fullcalendar#5024]
### Feature Requests
I've been trying to find an answer for this, and so far everything I've found indicates that it is currently impossible for fullcalendar to get event attachments from google calendar. However, all of the answers are from several years ago, and their reasoning is that the attachments feature is still in beta. 

Has there been any progress on this since than? according to the google [calendar APi documentation](https://developers.google.com/calendar/v3/reference/events), information about attachments can be sent with requests, so it should be possible to get that info using fullcalendar, right?

In short: is it possible to add support for google calendar attachments? 

Regardless, thanks for all the work thats been put into this, it's a great calendar framework!

**Comment by acerix** [comment]
I can't easily test this, but you might find the attachments in `extendedProperties`.

https://fullcalendar.io/docs/google-calendar#extended-properties

Otherwise that would be a new feature, so for now you would need to query the API manually to get them.

**Comment by pearcemichal** [comment]
The attachment does not appear to be in the extended properties.

**Comment by shudas** [comment]
Hi not sure if this is still actively being looked at but I needed to get this info for a project I was working on as well. I found a simple solution that would put it on the extendedProperties on the Events https://github.com/fullcalendar/fullcalendar/pull/6158

**Comment by arshaw** [comment]
I've merged https://github.com/fullcalendar/fullcalendar/pull/6158 and will release it in the next release. thanks!

**Comment by mtelloivan** [comment]
Hola necesito darme de baja de la lista de github y no me deja loguearme en
la web me dice que mi correo ***@***.*** es inexistente,

[image: image.png]

Gracias

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by arshaw** [comment]
this is released!
https://github.com/fullcalendar/fullcalendar/releases/tag/v5.6.0
